marshal/bin/agent.py

```diff
@@ -263,7 +263,9 @@ def main():
             rootfs_s3_etag = None
 
             for artifact in manifest["artifacts"]:
-                if artifact["type"] == "application/vnd.cray.image.rootfs.squashfs":
+                if artifact["type"] == "application/vnd.cray.image.rootfs.squashfs" and \
+                    'link' in artifact and artifact['link'] is not None and \
+                    'path' in artifact['link'] is not None and 'etag' in artifact['link'] is not None:
```

Comment:
This line look weird to me.   When I open this code in visual studio code with PEP8 support it shows https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/bad-chained-comparison.html

Reply:
@aasha-hpe fixed this.